src/elements/hx-search-control/index.js

```diff
@@ -0,0 +1,77 @@
+import { HXFormControlElement } from '../../interfaces/HXFormControlElement/index.js';
+
+/**
+ * Defines behavior for the `<hx-select-control>` element.
+ *
+ * @extends HXFormControlElement
+ * @hideconstructor
+ * @since 0.16.0
+ */
+export class HXSearchControlElement extends HXFormControlElement {
+    /** @override */
+    static get is () {
+        return 'hx-search-control';
+    }
+
+    $onCreate () {
+        this._onControlInput = this._onControlInput.bind(this);
+        this._onResetClick = this._onResetClick.bind(this);
+    }
+
+    $onConnect () {
+        this._showHideReset(this.controlElement);
+
+        this._btnReset.addEventListener('click', this._onResetClick);
+        this.controlElement.addEventListener('input', this._onControlInput);
+    }
+
+    $onDisconnect () {
+        this._btnReset.removeEventListener('click', this._onResetClick);
+        this.controlElement.removeEventListener('input', this._onControlInput);
+    }
+
+    /**
+     * Fetch the first `<input type="search">` descendant
+     *
+     * @override
+     * @readonly
+     * @type {?HTMLInputElement}
+     */
+    get controlElement () {
+        return this.querySelector('input[type="search"]');
+    }
+
+    /** @private */
+    get _btnReset () {
+        return this.querySelector('button.hxClear');
```

Comment:
@CITguy, what can/should we do (if anything) if `<button>` does not have `class="hxClear"` set?

Reply:
Don't do anything. The component won't have reset functionality if the button is either missing from the DOM or if it is missing the `hxClear` class.  It is up to the consumer to ensure that their markup matches what is documented.

Reply:
Speaking of which... we need documentation for the `<hx-search-control>` element (specifically the "Permitted Children").

Reply:
Ok, I will update the `<hx-search-control>` documentation in Phase II.